src/common/feature-flags.ts

```diff
@@ -118,6 +119,14 @@ export function getAllFeatureFlagDetails(): FeatureFlagDetail[] {
             isPreviewFeature: true,
             forceDefault: false,
         },
+        {
+            id: FeatureFlags.saveAndLoadAssessment,
+            defaultValue: false,
+            displayableName: 'Save / Load Assessment',
+            displayableDescription: 'Enables saving and loading assessments.',
+            isPreviewFeature: true,
```

Comment:
`isPreviewFeature` should be `false` here so the feature flag does not show up as an option in the Preview Features panel by default (we don't want users to see this until the feature is ready for them to try out).

Run `insightsFeatureFlags.enableFeature('showAllFeatureFlags')` in the DevTools console for the background to make even private feature flags appear in the panel.

(Changing this will also make the preview-features E2E test pass; it's failing because the snapshot was not updated to include this new feature flag)

Reply:
Updated, thank you!